src/app/modules/side-menu/components/selected-items-counter-label/selected-items-counter-label.component.html

```diff
@@ -0,0 +1,11 @@
+<ion-label>
+  {{ "OBSERVATION_FILTER.OBSERVATION_TYPE_FILTER.TITLE" | translate }}
+  <span
+    >({{
+      selectedItemsCount == 0
+        ? ("OBSERVATION_FILTER.OBSERVATION_TYPE_FILTER.NUM_OF_SELECTED_NONE" | translate)
+        : ("OBSERVATION_FILTER.OBSERVATION_TYPE_FILTER.NUM_OF_SELECTED"
+          | translate: { selectedItemsCount: selectedItemsCount })
```

Comment:
Siden det nå er en generell funksjon for å vise antall valgte, passer det ikke å legge disse tekstene under "OBSERVATION_TYPE_FILTER".